localstorage-sync.html

```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+<html>
+    <head>
+        <title>Deriv</title>
+        <meta charset="utf-8" />
+        <meta name="referrer" content="origin" />
+        <script>
+            window.addEventListener('message', function (message) {
+                var allowed_origins = [
+                    'https://deriv.app',
+                    'https://staging.deriv.app',
+                    'https://app.deriv.com',
+                    'https://app.deriv.me',
+                    'https://app.deriv.be',
+                    'https://staging-app.deriv.com',
+                    'https://staging-app.deriv.me',
+                    'https://staging-app.deriv.be',
+                    'https://test-app.deriv.com',
+                ];
+
+                if (allowed_origins.includes(message.origin)) {
+                    switch (message.data.key) {
+                        case 'active_loginid': {
+                            if (localStorage.getItem(message.data.key) !== message.data.value) {
+                                localStorage.setItem(message.data.key, message.data.value);
```

Comment:
we are setting this localStorage value but when/where should we use this?

Reply:
it's not completely implemented. for now I just want to make sure that we can recieve messages coming from deriv-app and update localstorage. next step is to setup these two things:
1- using localStorage to check if the user is logged in and also what is the active login id and based on that decide whether the user should see p2p page or redirect back to deriv-app/oauth page

2- implement the logic for sending authentication data back to other aps like deriv-app so they can be in sync with p2p